modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -3,208 +3,125 @@ ifndef::env-site,env-github[]
-
-== Generate a Certificate-Key Pair
-You need a certificate-key pair for the internal load balancer in Runtime Fabric. If you have one already, both files must be of the .pem type, your keystore must have a passcode, and your common name should match the domain you use in your Runtime Fabric. 
-
-RSA keys are the most common type of keys. RSA keys of 2K length offer the best compromise between security and performance. 
 [NOTE]
-RSA keys larger than 2K protect against brute force cracking and are appropriate for certificates that have expirations of many years. However, whenever key length is doubled, for example, from 2k to 4k, performance is reduced by a factor greater than 6.
- 
-ECDSA keys are also supported. In most cases, ECDSA doubles the performance of a 2K RSA key. Supported curves are:
-
-* secp521r1 (P-521)
-* secp384r1 (P-384)
-* secp256r1 (also known as prime256v1 (P-256))
-
-Use the following steps to generate a certificate-key pair for evaluation purposes.
-
-. Run the following command on your machine to generate a certificate-key pair:
-+
-----
-openssl req -x509 -newkey rsa:2048 -keyout key.pem -out cert.pem -days 365
-----
-
-. Type a passphrase for your key.
-. Complete the requested information. When asked for a common name, supply the domain to be used in your Runtime Fabric.
-
-If you use a wildcard, for example, `*.example.com` in your common name, your application URLs use the following format: `{app-name}.example.com`. Otherwise, your application URLs use the format `example.com/{app-name}`.
+It can take up to 5 minutes for any permissions you add to propagate.
 
+== Step 2: Configure Inbound Traffic Using Runtime Manager
 
-== Create a TLS Context
-
-Follow these steps to include your certificate-key pair in a TLS context and store it in Secrets Manager. This will be used later to encrypt inbound traffic to your Runtime Fabric.
-
-. Navigate to Secrets Manager from the Anypoint Platform homepage.
+. Navigate to Runtime Manager and select *Runtime Fabric*.
+. Select the name of your Runtime Fabric to open the management page.
+. In the *Associated Environments* tab, ensure your Runtime Fabric is associated with the environments in which applications are to be deployed.
+. Select *Inbound Traffic* and enable the *Enable inbound traffic* toggle.
 +
 [NOTE]
-Make sure you’re in the right business group and select your Runtime Fabric environment in the sidebar on the left.
+If you disable the Inbound Traffic tab, changes can take several minutes to complete.
 
-. Select the *Create Secret Group* button.
-. Under *General*, type the name of your secret group.
-. Select *Save*.
-. Select *Keystore* in the sidebar and create your keystore:
-.. Select *Add Keystore*.
+. In the *Basic Configuration* section, verify *Secure Port* is set to `443` and *Non Secure Port* is set to `80`.  The nonsecure port redirects to the secure port.
+. Select *Resource Allocation*.
```

Comment:
This step is at such a great distance from the step following it that I had to scroll up and down a few times to understand the information hierarchy.
Please create a JIRA for yourself to review the organization of this topic or topic set at some future time, keeping in mind that tasks should not include so much conceptual information nor comprise a complicated nested hierarchy.